docs/UserGuide.md

```diff
@@ -478,13 +489,18 @@ ________________________________________________________________________________
 help:              shows the list of commands available at the topic level
 add abc; def:      adds a flashcard with question 'abc' and answer 'def' in the current topic
 list:              shows the list of all flashcards in the current topic
+list all:          shows the tree of all subjects, topics, tasks and flashcards
 delete 1:          deletes the 1st flashcard in the list
 exit:              exits the topic to return to the subject level, where you can work with tasks and topics
 ________________________________________________________________________________________________________________________
 ```
-#### 4.4.2 `add` - Add flashcard
+WARNING: Do not add extra spaces after or before help.
```

Comment:
imo note instead of warning would be better here, since nothing bad will happen even if they add extra spaces in this case